ios/browser/ui/webui/brave_wallet/brave_wallet_page_ui.mm

```diff
@@ -0,0 +1,179 @@
+#include "brave/ios/browser/brave_wallet/brave_wallet_ipfs_service_factory.h"
+#include "brave/ios/browser/brave_wallet/brave_wallet_service_factory.h"
+#include "brave/ios/browser/brave_wallet/swap_service_factory.h"
+#include "brave/ios/browser/ui/webui/brave_wallet/wallet_common_ui.h"
+#include "components/grit/brave_components_resources.h"
+#include "components/grit/brave_components_strings.h"
+#include "ios/chrome/browser/shared/model/browser_state/chrome_browser_state.h"
+#import "ios/web/public/web_state.h"
+#include "ios/web/public/webui/url_data_source_ios.h"
+#import "ios/web/public/webui/web_ui_ios.h"
+#import "ios/web/public/webui/web_ui_ios_data_source.h"
+#import "ios/web/public/webui/web_ui_ios_message_handler.h"
+#include "ui/base/accelerators/accelerator.h"
+#include "ui/base/webui/web_ui_util.h"
+
+namespace {
+
+web::WebUIIOSDataSource* CreateAndAddWebUIDataSource(
+    web::WebUIIOS* web_ui,
+    const std::string& name,
+    const webui::ResourcePath* resource_map,
+    std::size_t resource_map_size,
+    int html_resource_id) {
+  web::WebUIIOSDataSource* source = web::WebUIIOSDataSource::Create(name);
+  web::WebUIIOSDataSource::Add(ChromeBrowserState::FromWebUIIOS(web_ui),
+                               source);
+  source->UseStringsJs();
+
+  // Add required resources.
+  source->AddResourcePaths(base::make_span(resource_map, resource_map_size));
+  source->SetDefaultResource(html_resource_id);
+  return source;
+}
+
+}  // namespace
+
+BraveWalletPageUI::BraveWalletPageUI(web::WebUIIOS* web_ui, const GURL& url)
+    : web::WebUIIOSController(web_ui, url.host()) {
+  web::WebUIIOSDataSource* source;
+  if (url.path() == "/swap" || url.path() == "/send") {
+    source = CreateAndAddWebUIDataSource(web_ui, url.host(),
+                                         kBraveWalletIosContainerGenerated,
+                                         kBraveWalletIosContainerGeneratedSize,
+                                         IDR_BRAVE_WALLET_IOS_CONTAINER_HTML);
+  } else {
+    NOTREACHED() << "Failed to find page resources for:" << url.path();
+  }
+
+  for (const auto& str : brave_wallet::kLocalizedStrings) {
+    std::u16string l10n_str =
+        brave_l10n::GetLocalizedResourceUTF16String(str.id);
+    source->AddString(str.name, l10n_str);
+  }
+
+  source->AddBoolean("isAndroid", false);
+  source->AddBoolean("isIOS", true);
+
+  source->AddString("braveWalletLedgerBridgeUrl", kUntrustedLedgerURL);
+  source->AddString("braveWalletTrezorBridgeUrl", kUntrustedTrezorURL);
+  source->AddString("braveWalletNftBridgeUrl", kUntrustedNftURL);
+  source->AddString("braveWalletLineChartBridgeUrl", kUntrustedLineChartURL);
+  source->AddString("braveWalletMarketUiBridgeUrl", kUntrustedMarketURL);
+  source->AddBoolean(brave_wallet::mojom::kP3ACountTestNetworksLoadTimeKey,
+                     base::CommandLine::ForCurrentProcess()->HasSwitch(
+                         brave_wallet::mojom::kP3ACountTestNetworksSwitch));
+
```

Comment:
`braveWalletLedgerBridgeUrl` and `braveWalletTrezorBridgeUrl` are for hw wallets and seems don't needed for mobile platforms

Reply:
We'll fail to render with `Could not find value for braveWalletLedgerBridgeUrl/braveWalletTrezorBridgeUrl` when loading send/swap without these as they are accessed via `loadTimeData`. 

cc @josheleonard these are loaded when exporting constants in [ledger-messages.ts](https://github.com/brave/brave-core/blob/fd4114fb454a215f61a0402d36c9a4a2c2f79a13/components/brave_wallet_ui/common/hardware/ledgerjs/ledger-messages.ts#L20-L27) / [trezor-messages.ts](https://github.com/brave/brave-core/blob/fd4114fb454a215f61a0402d36c9a4a2c2f79a13/components/brave_wallet_ui/common/hardware/trezor/trezor-messages.ts#L17-L19), what if first check `loadTimeData` for isIOS / isAndroid and return empty string? Or is there a better way we could isolate these constants?

Reply:
> We'll fail to render with Could not find value for braveWalletLedgerBridgeUrl/braveWalletTrezorBridgeUrl when loading send/swap without these as they are accessed via loadTimeData.

Let's keep them here so it's simple.

Acutally these are constants and don't need to be passed with loadTimeData. Mojom constants should be enough